install/mobile-dashboard/install.yml

```diff
@@ -0,0 +1,28 @@
+# in install/example/install.yml
+
+# Defined by the author, must be unique
+id: mobile-dashboard-install
+
+# A human readable name
+name: Mobile Dashboard Install Plan
+
+# Used as a heading during the install process
+title: Mobile Dashboard Install Plan
+
+description: |
+  Installs the Mobile Dashboard.
+
+target:
+  # type can be agent | integration | on_host_integration | unknown
+  type: unknown
+  # destination can be application | cloud | host | kubernetes | unknown
+  destination: unknown
+  # os can be darwin | linux | windows
+  os:
+    - linux
+    - windows
+
+install:
+  mode: link
+  destination:
+    url: https://docs.newrelic.com/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#get-started
```

Comment:
While these docs do relate to the dashboard, generally it's good to provide a link out to the instructions on how to get the data into New Relic. Perhaps we could add a link to the mobile agent installation docs?

Reply:
Done!